src/routes/events.js

```diff
@@ -84,7 +84,7 @@ router.put('/register/:eventId/', auth, async (req, res) => {
 			'users'
 		);
 		if (registeredUsers.users.includes(userId)) {
-			return res.status(400).json({ msg: 'User Already Registered' });
+			return res.status(400).json({ code: 'EVENTS_REGISTER_REDUNDANT', msg: 'User Already Registered' });
```

Comment:
Make this code something else!

Reply:
Done.
Ref: 6a2ddc7ac52071efdfeddcaf94fc721f8e70d1ff